doc/openocd.texi

```diff
@@ -10902,6 +10902,14 @@ Perform a 32-bit DMI read at address, returning the value.
 Perform a 32-bit DMI write of value at address.
 @end deffn
 
+@deffn {Command} {riscv dm_read} address
+Perform a 32-bit DM read at address, returning the value.
+@end deffn
+
+@deffn {Command} {riscv dm_write} address value
+Perform a 32-bit DM write of value at address.
+@end deffn
+
```

Comment:
Here I'd like to suggest little changes in the documentation:

The `riscv dm_read` and `riscv dm_write` commands should go first, because they are expected to be used more often.

The text should be updated to clearly show the difference between these commands. My proposal is below. 

@timsifive Does that wording work for you, too?

```
The following commands allow for direct low-level access to the registers 
of the Debug Module (DM) via the Debug Module Interface (DMI) bus. 
These commands can be used to interact with custom debug features.

@deffn {Command} {riscv dm_read} reg_address
Perform a 32-bit read from a DM register at reg_address, returning the value. 
The reg_address should be relative to the base DMI address of the DM. 
OpenOCD adds the DM base address (dbgbase) automatically.
@end deffn

@deffn {Command} {riscv dm_write} reg_address value
Perform a 32-bit write to a DM register at reg_address. 
The reg_address should be relative to the base DMI address of the DM. 
OpenOCD adds the DM base address (dbgbase) automatically.
@end deffn

Note: The DM base address (dbgbase) is typically zero unless there 
are multiple DMs on the DMI bus.

@deffn {Command} {riscv dmi_read} address
Perform a 32-bit read from the given DMI address, returning the value.
@end deffn

@deffn {Command} {riscv dmi_write} address value
Perform a 32-bit write to the given DMI address.
@end deffn
```

Reply:
I agree, it looks easier to read now

Reply:
Seems fine to me.